scenarioo-docu-generation-example/src/test/resources/config-for-demo/config.xml

```diff
@@ -57,6 +57,18 @@
         </branchAlias>
     </branchAliases>
 	<comparisonConfigurations>
+        <comparisonConfiguration>
+            <name>To Develop</name>
+            <baseBranchName>scenarioo-feature-*</baseBranchName>
```

Comment:
Shouldn't this be `.*` instead of just `*`?

Reply:
Yes, should! Thought I did that -will check ...